data/human/news.txt

```diff
@@ -1990,6 +2093,75 @@ news "conspiracy theorist"
 
 
 
+news "players crew member"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	to show
+		"flagship crew" > 30
+	name
+		word
+			"Some crew member"
```

Comment:
```suggestion
			"Crew member"
```

"Some" doesn't add much here, and doesn't fit the rest of the phrases.

Reply:
Agreed. Crew member works better.